### DISCUSSION TITLE
Is there any plan to add Pinia Icon to popular icon sets?

### DISCUSSION THREAD

**Discussion by JavascriptMick** [vuejs/pinia#2165] [Help and Questions]
I'm using Pinia and wanted to showcase it as part of my stack by including the icon.  I was unable to find the icon in popular icon sets (I'm using vue icon and iconify...)   https://icon-sets.iconify.design/?query=pinia

Is there any plan to put the Pinia icon in some of these icon sets so it can be accessed through these tools?

**Comment by posva** [comment]
It would be nice! It would help me if people opened issues or PR in the relevant sites. I'ven seen some opened already but I cannot handle this all by myself.

**Reply by posva** [reply]
Found one the issues: https://github.com/simple-icons/simple-icons/issues/7498
